event/handler.go

```diff
@@ -281,6 +283,56 @@ var SortFilter = func(ctx context.Context, handler *ExportHandler, event *Filter
 	}
 }
 
+var CreateFilterForAll = func(ctx context.Context, handler *ExportHandler, event *FilterSubmitted, isPublished bool) (*observation.DimensionFilters, error) {
+	// Get the names of the dimensions for the DatasetID
+	dimensions, err := handler.datasetAPICli.GetVersionDimensions(ctx,
+		"", // userAuthToken
+		handler.serviceAuthToken,
+		"", // collectionID
+		event.DatasetID, event.Edition, event.Version,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	var dbFilterDimensions []*observation.Dimension
+
+	for _, dim := range dimensions.Items {
+		// Get the names of the options for a dimension
+		opts, err := handler.datasetAPICli.GetOptionsInBatches(ctx,
+			"", // userAuthToken
+			handler.serviceAuthToken,
+			"", // collectionID
+			event.DatasetID, event.Edition, event.Version, dim.Name,
+			100,
```

Comment:
maybe these values should be configurable? (or maybe we can do it later if we only want to test the performance difference at this point)

Reply:
We only need to get the new feature working at this point.
Any further performance work will have to be a new ticket.